accounts/fixtures/email_type.json

```diff
@@ -0,0 +1 @@
+[{"model": "accounts.emailtype", "pk": 1, "fields": {"description": "Email notification sent when user's sound is selected as random sound of the day", "name": "random_sound", "display_name": "Selected sound as random sound of the day"}}, {"model": "accounts.emailtype", "pk": 2, "fields": {"description": "Email notification sent when user's receives a private message", "name": "private_message", "display_name": "New private message received"}}, {"model": "accounts.emailtype", "pk": 4, "fields": {"description": "Email notification sent when user's receives a comment on a sound", "name": "new_comment", "display_name": "New comment on sound"}}]
```

Comment:
I guess this fixture is generated by django's `manage.py dumpdata app.Model > fixture.json` command.  Given that this fixture is small, using `--indent 4` as parameter should do the trick and make it more readable.